rackcity/api/serializers/log_serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from rackcity.models import Log
+
+
+class LogSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Log
+        fields = (
+            'id',
+            'date',
+            'log_content',
+            'user',
+            'related_assets',
+            'related_model',
```

Comment:
Is it model singular and assets plural because network connections can be between assets but everything else would only be one resource?

Reply:
Would you not log actions on racks or datacenters?

Reply:
Also I trust you have a plan for how you're displaying the logs and how you're linking to models/asset and how that all relates to how you serialize this

Reply:
1. yes, network connections are the ONLY reason there would ever be more than one asset linked. model doesn't have anything like that. 
2. yes logging actions on these, but the related field will be used for linking, and we do NOT need to link racks/datacenters/users because the link should bring you to the detail view, and only assets and models have detail views
3. TRUST FALL